### PR TITLE
Rework colors to a more consistent set of hues

### PR DESCRIPTION
**Pull request by jazeee**
Convert table headers to match the sheet theme.
Adjust styles to clean up some anomalies.
Clean up some minor styling differences.
Update background URL to match this repo.

Examples:

image
image
image

### CODE REVIEW

File: bin/main.css [hunk 1]
```diff
@@ -47,7 +47,7 @@ button.actionRollButton {
 }
 
 /* Change the character for dice roll buttons to be
-   the two joined squares character and give them the blue background */
+   the two joined squares character and add styles */
 button.btn.ui-draggable[type=roll],
```
Comment (jazeee) [inline bin/main.css:51]: Generally, I'd review the source files below...

**Review by johnlamping — APPROVED** [review]
Myccorizae's revenge for the win!
This looks great.
There are a couple of small issues for a follow-up: The CP and TP labels don't line up vertically with the values. Also, they would probably look better if they were bold, to match the values.

**Comment by jazeee** [comment]
Agreed, I want to rework the top area, and also rework other tabs to use similar table styles.
Follow ups will be coming.

**Comment by jazeee** [comment]
https://github.com/epic-power-rpg/roll20/issues/30 captures the label issues.

**Pull request merged**